### DISCUSSION TITLE
500 SERVER ERROR

### DISCUSSION THREAD

**Discussion by Leo6Leo** [yhype/yhype-feedback#17] [Q&A]
When I was trying to use Github to login yhype, it showed 500 SERVER ERROR.
<img width="1388" alt="CleanShot 2023-01-10 at 00 35 42@2x" src="https://user-images.githubusercontent.com/36619969/211470059-97d776ce-868c-470b-8e8f-c28329c1cbd5.png">

**Comment by antonkomarev** [comment]
@Leo6Leo is it still 500? I see it is working well

**Reply by antonkomarev** [reply]
@Leo6Leo are you logged in or as guest?

**Reply by Leo6Leo** [reply]
I just openped https://yhype.me/github/accounts/<my user name>, and it works right now!

**Reply by Leo6Leo** [reply]
logged in state. The issue appeared last night, and it was my first time using Github to login.

**Reply by antonkomarev** [reply]
Looks like temporary issues

**Reply by Leo6Leo** [reply]
Opps, it seems like a problem. It seems like every day at night around 20:00-21:00(EST), the issue will appear again. And in the morning, it will go back to normal. Right now(20:55 EST, Jan. 11), the 500 server error just shows up again when I visit https://yhype.me/github/accounts/

**Reply by aeiouaeiouaeiouaeiouaeiouaeiou** [reply]
Is there any chance of it working again? 🙃

**Reply by antonkomarev** [reply]
Fixed it. Thanks

**Reply by Leo6Leo** [reply]
I think the issue is still there Anton @antonkomarev . The 500 server error shows up again. Right now is 18:00 EST.

**Reply by antonkomarev** [reply]
Sorry for that. We've made some changes. I hope now it wouldn't fall every day...

**Comment by albertopasqualetto** [comment]
It happens frequently

**Comment by antonkomarev** [comment]
We started to work on infrastructure transparency and stability. Huge amount of daily data streams. Need to understand where is the bottleneck. It's pretty long way, but in the end the service will become much more stable.

**Comment by antonkomarev** [comment]
We found 2 issues and resolved them. Should not return 500 now. 2 days without issues. Thanks for reporting!
